runtime/doc/vimdiff.1

```diff
@@ -1,4 +1,4 @@
-.TH VIMDIFF 1 "2001 March 30"
+.TH VIMDIFF 1 "2021 June 13"
```

Comment:
why just the date change here?

Reply:
I based it on the date of the last change ("commit"). It is possible that they just forgot to change the date.
https://github.com/vim/vim/commit/2346a6378483c9871016f9fc821ec5cbea638f13#diff-2682a2bf53e9ab4a0692e5c69970484396e3f1a06f0e2be4c615a845d47c1a60

Reply:
Okay I see